sorc/grid_tools.fd/regional_esg_grid.fd/pmat5.f90

```diff
@@ -1,34 +1,30 @@
 !> @file
-!! @author R. J. Purser @date 1996 
-!!
 !! Handy geographical transformations
-!!
 !! DEPENDENCIES
```

Comment:
Take out sections on dependencies. Fortunately, doxygen will figure those out for us.